testsuite/tests/libratest/smoke_test.rs

```diff
@@ -558,7 +558,53 @@ fn test_full_node_basic_flow() {
         );
     }
 
-    // writes through full node AC are disabled for now
-    let mint_result = full_node_client.mint_coins(&["mintb", "0", "1"], true);
-    assert!(mint_result.is_err());
+    // mint from full node and check both validator and full node have correct balance
+    let mint_result = full_node_client.mint_coins(&["mintb", "0", "10"], true);
+    assert!(mint_result.is_ok());
+    assert_eq!(
+        Decimal::from_f64(17.0),
+        Decimal::from_str(&full_node_client.get_balance(&["b", "0"]).unwrap()).ok()
+    );
+    assert_eq!(
+        Decimal::from_f64(17.0),
+        Decimal::from_str(&validator_ac_client.get_balance(&["b", "0"]).unwrap()).ok()
+    );
+
+    // mint from validator and check both nodes have correct balance
+    let mint_result = validator_ac_client.mint_coins(&["mintb", "1", "10"], true);
+    assert!(mint_result.is_err()); // validator and full node client numbers are out of sync, so error is expected. Needs to pass on retry.
+
+    validator_ac_client
+        .mint_coins(&["mintb", "1", "10"], true)
+        .unwrap();
+    assert_eq!(
+        Decimal::from_f64(14.0),
+        Decimal::from_str(&validator_ac_client.get_balance(&["b", "1"]).unwrap()).ok()
+    );
+
+    sleep(time::Duration::from_millis(100)); // wait for full node to sync up
```

Comment:
explicit sleep here feels dirty. Can we instead block on some cli command
e.g.  I think you can call explicitly `wait_for_transaction`